rust/prql/include/prql.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+extern "C" {
+
+    int to_json(const char * query, char * out);
+
+    int to_sql(const char * query, char * out);
```

Comment:
The name is too generic, consider using something like `prql_to_sql` and `prsql_to_json`

Reply:
I used the `prql_` prefix as you suggested.